floating-group.lisp

```diff
@@ -135,12 +135,17 @@
   (float-window-align window)
   (focus-window window))
 
+(defmethod group-add-window (group (window float-window) &key &allow-other-keys)
```

Comment:
I'd like to refactor some of the code to remove the use of multimethods where it doesn't seem entirely appropriate. What do you think? See: https://github.com/stumpwm/stumpwm/issues/300